docs/.eleventy.js

```diff
@@ -179,14 +179,63 @@ module.exports = function(eleventyConfig) {
         `.trim();
     }
 
+    /**
+     * Encodes text in the base 64 format used in playground URL params.
+     * @param {string} text Text to be encoded to base 64.
+     * @see https://github.com/eslint/eslint.org/blob/1b2f2aabeac2955a076d61788da8a0008bca6fb6/src/playground/utils/unicode.js
+     * @returns {string} The base 64 encoded equivalent of the text.
+     */
+    function encodeToBase64(text) {
+        /* global btoa -- It does exist, and is what the playground uses. */
+        return btoa(unescape(encodeURIComponent(text)));
+    }
+
+    /**
+     * Creates markdownItContainer settings for a playground-linked codeblock.
+     * @param {string} name Plugin name and class name to add to the code block.
+     * @returns {[string, object]} Plugin name and options for markdown-it.
+     */
+    function withPlaygroundRender(name) {
+        return [
+            name,
+            {
+                render(tokens, index) {
+                    if (tokens[index].nesting !== 1) {
+                        return "</div>";
+                    }
+
+                    // See https://github.com/eslint/eslint.org/blob/ac38ab41f99b89a8798d374f74e2cce01171be8b/src/playground/App.js#L44
+                    const parserOptions = tokens[index].info?.split("correct ")[1]?.trim();
+                    const { content } = tokens[index + 1];
+                    const state = encodeToBase64(
+                        JSON.stringify({
+                            ...(parserOptions && { options: { parserOptions: JSON.parse(parserOptions) } }),
+                            text: content
+                        })
+                    );
+                    const prefix = process.env.CONTEXT && process.env.CONTEXT !== "deploy-preview"
```

Comment:
aren't both same? 
```suggestion
                    const prefix = process.env.CONTEXT !== "deploy-preview"
```

Reply:
I'm not 100% on this particular repo's env contexts 😄 but based on this...

https://github.com/eslint/eslint/blob/077ca1e534d746e84dc4119affaad2ba152bc73c/docs/.eleventy.js#L520-L521

...I was assuming that `process.env.CONTEXT` not existing is different from it existing and not being `"deploy-preview"`.

Reply:
gotcha 👍🏻 I just verified local did not have `CONTEXT`. resolving this too.